.github/workflows/CI.yml

```diff
@@ -35,25 +35,19 @@ jobs:
         node: [18.x, 20.x]
         include:
           - os: ubuntu-latest
-            node: "16.0.x"
-            eslint: "8.x"
-          - os: ubuntu-latest
-            node: "16.x"
-            eslint: "8.x"
-          - os: ubuntu-latest
-            node: "19.x"
+            node: "21.x"
             eslint: "8.x"
           - os: ubuntu-latest
             node: "20.x"
-            eslint: "7.0.x"
+            eslint: "9.0.0-alpha.0"
```

Comment:
**suggestion:** It may be worth adding this to line 34

Reply:
npm does not work well with prereleases, e.g. `eslint@9.0.0-alpha.0`, hope it can support something like `npm install --allow-prereleases`.  🧐